File: src/app/Watchlist.ts
import { Filme } from "./Filme";
import { Usuario } from "./Usuario";

export class Watchlist{
    id: number = 0;
    filmeId: number = 0;
    filme: Filme | null = null;
    usuarioId: number = 0;
    usuario: Usuario | null = null;
}

File: Data/Models/Watchlist.cs
namespace TrabalhoAPI.Data.Models
{
    public class Watchlist
    {
        public int Id { get; set; }
        public int FilmeId { get; set; }
        public Filme? Filme { get; set; }
        
        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }

        
    }
}